main/src/com/google/refine/util/PatternSyntaxExceptionParser.java

```diff
@@ -0,0 +1,127 @@
+/*
+
+Copyright 2010, Google Inc.
```

Comment:
Why are you copyrighting as Google instead of yourself ?

Reply:
Lazy just copying existing files as a template - will amend!